src/test/java/org/fundacionjala/coding/rodrigo/OcrBanKTest.java

```diff
@@ -0,0 +1,76 @@
+package org.fundacionjala.coding.rodrigo;
+
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+
+/**
+ *
+ */
+public class OcrBanKTest {
+    /**
+     * Test verifying two number.
+     */
+    @Test
+    public void testConvertTwoNumber() {
+        String s1 = " _    \n";
+        s1 = s1.concat("  ||_|\n");
+        s1 = s1.concat("  |  |");
+        assertEquals("74", OcrBank.convertOcrToNumber(s1));
+    }
+
+    /**
+     * Verifying some acc1.
+     */
+    @Test
+    public void testConvertAcc1() {
+        String s2 = "    _  _     _  _  _  _  _ \n";
+        s2 = s2.concat("  | _| _||_||_ |_   ||_||_|\n");
+        s2 = s2.concat("  ||_  _|  | _||_|  ||_| _|");
+        assertEquals("123456789", OcrBank.convertOcrToNumber(s2));
+        assertEquals("123456789", OcrBank.verificationIllErr(s2));
+    }
+
+    /**
+     * Verifying some acc2.
+     */
+    @Test
+    public void testConvertAcc2() {
+        String s3 = " _  _  _  _  _  _  _  _    \n";
+        s3 = s3.concat("| || || || || || || ||_   |\n");
+        s3 = s3.concat("|_||_||_||_||_||_||_| _|  |");
+        assertEquals("000000051", OcrBank.convertOcrToNumber(s3));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "000000051" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)